app/src/main/java/com/cornellappdev/android/eatery/model/MealMenuModel.java

```diff
@@ -0,0 +1,63 @@
+package com.cornellappdev.android.eatery.model;
+
+import java.io.Serializable;
+import java.util.ArrayList;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Map.Entry;
+import org.json.JSONArray;
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class MealMenuModel implements Serializable {
+
+  private Map<String, List<String>> menu = new HashMap<>();
+
+  public void addItem(String category, String item) {
+    if (menu.get(category) == null) {
+      menu.put(category, new ArrayList<>());
+    }
+
+    menu.get(category).add(item);
+  }
+
+  public List<String> getItems(String category) {
+    return new ArrayList<>(menu.get(category));
+  }
+
+  public List<String> getCategories() {
+    return new ArrayList<>(menu.keySet());
+  }
+
+  public int getNumberOfCategories() {
+    return menu.keySet().size();
+  }
+
+  public List<String> getAllItems() {
+    List<String> items = new ArrayList<>();
+
+    for (List<String> val : menu.values()) {
+      items.addAll(val);
+    }
+
+    return items;
```

Comment:
same thing here, `for` can be inlined

Reply:
Google style guidelines.

Reply:
https://google.github.io/styleguide/javaguide.html#s4.1.1-braces-always-used